src/cache/shared.ts

```diff
@@ -2,6 +2,7 @@ import { IdToken, User } from '../global';
 
 export const CACHE_KEY_PREFIX = '@@auth0spajs@@';
 export const CACHE_KEY_ID_TOKEN_SUFFIX = '@@user@@';
+export const USER_CACHE_KEY = 'idToken';
```

Comment:
Should we just use `@@user@@` as well? or `@@idToken@@`. It doesn't matter tbh, but would look more consistent. Or just keep `USER_CACHE_KEY` in `Auth0Client`, as it's not realy a reused key outside of the Client, that would also solve the conflict in value (@@ vs no @@). What do you think?

Reply:
I was thinking to reuse `@@user@@` and the only reason I didn't want because of the `SUFFIX` in the variable name. That's not really that strong a reasoning so I think for consistency yeah it'd be good to use that

Reply:
Or `export const USER_CACHE_KEY = '@@user@@';` could do, same value but doesnt need to be the exact same variable imho.